content/release-10-18-0/whatsnew-10-18-0.md

```diff
@@ -7,3 +7,11 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+
+
+### Authentication
+
+#### Authentication with OAuth2 access tokens from authorization servers
+
+Access tokens from third party authorization servers can be used to authenticate in Cumulocity IoT. The SSO configuration is extended to allow the configuration of such authentication. Configuring this authentication,
```

Comment:
```suggestion
It is now possible to use access tokens from third-party authorization servers to authenticate in Cumulocity IoT. The SSO configuration has been extended accordingly. Configuring this authentication type,
```

Reply:
> 
On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).